sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -3394,31 +3394,31 @@ gather(AccessorT acc, OffsetSimdViewT byte_offsets, PropertyListT props = {}) {
 ///
 /// template <typename T, int N, int VS = 1, typename AccessorTy,
 /// typename OffsetT, typename PropertyListT = empty_properties_t>
-/// void scatter(T *p, simd<OffsetT, N / VS> byte_offsets, simd<T, N> vals,
-/// 	simd_mask<N / VS> mask, PropertyListT props = {}); // (acc-sc-1)
+/// void scatter(AccessorTy acc, simd<OffsetT, N / VS> byte_offsets, simd<T, N>
+/// vals, 	simd_mask<N / VS> mask, PropertyListT props = {}); // (acc-sc-1)
```

Comment:
nit: the indentations look odd here and in other places:
```suggestion
/// void scatter(AccessorTy acc, simd<OffsetT, N / VS> byte_offsets,
///              simd<T, N> vals, simd_mask<N / VS> mask,
///              PropertyListT props = {});                        // (acc-sc-1)
```

Reply:
should be fixed in https://github.com/intel/llvm/pull/12670/commits/c0ff4b7f8a89607245cee65423723ec538430f8e